libs/deps/merch-card.js

```diff
@@ -38,6 +38,7 @@ import{html as n,LitElement as T}from"/libs/deps/lit-all.min.js";import{css as u
     .top-section {
         display: flex;
         justify-content: flex-start;
+        flex-flow: wrap;
```

Comment:
Please pull the latest from tacocat.js and rebuild the dependency again. This change would reopen a ticket already closed.